jcli/src/jcli_app/utils/open_api_verifier.rs

```diff
@@ -0,0 +1,146 @@
+use openapiv3::{OpenAPI, PathItem, ReferenceOr};
+use reqwest::Request;
+use std::env;
+use std::fs::File;
+use std::io;
+
+pub struct OpenApiVerifier(OpenApiVerifierMode);
+
+enum OpenApiVerifierMode {
+    AcceptAll,
+    Verify(OpenApiVerifierImpl),
+}
+
+struct OpenApiVerifierImpl {
```

Comment:
Let's not use "impl" in type names. `VerifierState`?

Reply:
It's not quite state, more like implementation of the mode. I think I quite often see `...Inner` in Rust ecosystem, would `OpenApiVerifierInner` be ok? Why not `Impl`? Too much like if it was implementor of an `OpenApiVerifier` interface?

Reply:
I agree with @mzabaluev

Reply:
I've renamed it to `pub OpenApiVerifier` -> `VerifierMode` -> `Verifier`, it's less verbose and names without `OpenApi` prefix are private anyway, so their OpenApi context is always clear